docs/src/idiomatic/assignments.md

```diff
@@ -0,0 +1,129 @@
+# Idiom 1: Update state variables with assignments
+For each TLA+ variable `x` defined in a specification, Apalache requires that every possible  transition defines an assignment to `x'`.
```

Comment:
Can you also add a quick comparison to TLC? I think we are pretty close to how TLC does it right now.

Reply:
Also, the standard terminology for transitions in TLA+ is *action*. So shall we talk about actions instead of transitions in this document? In the end, we construct symbolic transitions. But we can also view them as a particular case of actions.

Reply:
The problem here is granularity. From Lamport's book: 
```
... is an ordinary mathematical formula, except that it contains primed as well as unprimed variables.  Such a formula is called an _action_.  An action is true or false of a step
```
In this sense, `x' = 1 \/ x' = 2` is an action, whereas, in our terminology, it describes two transitions. In Apalache, the assignment constraints must hold for transitions, not just for actions.

Reply:
Yes, that is why we have a special form of actions.